scripts/analysisTools/w_mass_13TeV/makeImpactsOnMW.py

```diff
@@ -37,13 +37,21 @@ def readNuisances(args, infile=None):
 
     print(f"Starting with file {infile} ...")
 
-    massNuisanceName = "WmassShift{s}MeV".format(s=int(args.prefitUncertainty))
+    #massNuisanceName = "WmassShift{s}MeV".format(s=int(args.prefitUncertainty))
+    massNuisanceName = ".*massShift.*{s}MeV$".format(s=int(args.prefitUncertainty))
     #massNuisanceName = "massShift{s}MeV".format(s=int(args.prefitUncertainty))
     valuesAndErrors = utilities.getFromHessian(infile,params=[massNuisanceName])
-    totalUncertainty = valuesAndErrors[massNuisanceName][1] - valuesAndErrors[massNuisanceName][0]
+    if len(valuesAndErrors) == 1:
+        masskey = list(valuesAndErrors.keys())[0]
+        totalUncertainty = valuesAndErrors[masskey][1] - valuesAndErrors[masskey][0]
+    else:
+        print(f"Warning: found more parameters matching expected mass expression: {valuesAndErrors.keys()}")
```

Comment:
raise exception rather than print+quit

Reply:
Ok, I will fix it here and other places where it appears